api/e2e/test/helpers_cluster_test.go

```diff
@@ -35,8 +35,13 @@ func newClusterClients(cfg *testConfig) (*TestClusterClients, error) {
 	var clusterCfg *rest.Config
 
 	if cfg.KubeconfigUseLocal {
+		var p string
```

Comment:
Super nit.
```
p := cfg.KubeconfigFilePath
if (p == "") {
        p = filepath.Join(homedir.HomeDir(), ".kube", "config")
}
```

Reply:
lol ya for some reason I was just in a rush to get things to work.

Reply:
its ok i nit people who do this as well

```go
if booleanCondition {
  return true
} else {
  return false
}
```